modules/ppcp-onboarding/src/Endpoint/LoginSellerEndpoint.php

```diff
@@ -147,7 +147,7 @@ public function handle_request(): bool {
 
 			$accept_cards    = (bool) ( $data['acceptCards'] ?? true );
 			$funding_sources = array();
-			if ( $this->settings->get( 'disable_funding' ) ) {
+			if ( $this->settings->has( 'disable_funding' ) && $this->settings->get( 'disable_funding' ) ) {
```

Comment:
Why `get` here? You simply need to replace `get` with `has`, it was supposed to be `has`, just a mistype.

Reply:
Done!